neuro_data/static_images/data_schemas.py

```diff
@@ -964,76 +892,19 @@ class Member(dj.Part):
-            ('21067-15-9', dict(animal_id=21067, session=15, scan_idx=9, preproc_id=0)),
-            ('20892-10-10', dict(animal_id=20892, session=10, scan_idx=10, preproc_id=0)),
-            ('20457-5-17', dict(animal_id=20457, session=5, scan_idx=17, preproc_id=0)),
-            ('20505-10-19', dict(animal_id=20505, session=10, scan_idx=19, preproc_id=0)),
-            ('20892-4-16', dict(animal_id=20892, session=4, scan_idx=16, preproc_id=0)),
-            ('21067-10-18', dict(animal_id=21067, session=10, scan_idx=18, preproc_id=0)),
-            ('21067-11-21', dict(animal_id=21067, session=11, scan_idx=21, preproc_id=0)),
-            ('21067-12-15', dict(animal_id=21067, session=12, scan_idx=15, preproc_id=0)),
-            ('21067-13-14', dict(animal_id=21067, session=13, scan_idx=14, preproc_id=0)),
-            ('21553-11-10', dict(animal_id=21553, session=11, scan_idx=10, preproc_id=0)),
-            ('20892-9-11', dict(animal_id=20892, session=9, scan_idx=11, preproc_id=0)),
-            ('21844-2-12', dict(animal_id=21844, session=2, scan_idx=12, preproc_id=0)),
-            ('22085-2-20', dict(animal_id=22085, session=2, scan_idx=20, preproc_id=0)),
-            ('22083-7-21', dict(animal_id=22083, session=7, scan_idx=21, preproc_id=0)),
-            ('22083-6-18', dict(animal_id=22083, session=6, scan_idx=18, preproc_id=0)),
-            ('22279-4-23', dict(animal_id=22279, session=4, scan_idx=23, preproc_id=0)),
-            ('22564-2-12', dict(animal_id=22564, session=2, scan_idx=12, preproc_id=0)),
-            ('22564-2-13', dict(animal_id=22564, session=2, scan_idx=13, preproc_id=0)),
-            ('22564-3-8', dict(animal_id=22564, session=3, scan_idx=8, preproc_id=0)),
-            ('22564-3-12', dict(animal_id=22564, session=3, scan_idx=12, preproc_id=0)),
-        ]
-        for group_id, (descr, key) in enumerate(selection):
-            entry = dict(group_id=group_id, description=descr)
-            if entry in self:
-                print('Already found entry', entry)
-            else:
-                with self.connection.transaction:
-                    if not (InputResponse() & key):
-                        ValueError('Dataset not found')
-                    self.insert1(entry)
-                    for k in (InputResponse() & key).fetch(dj.key):
-                        k = dict(entry, **k)
-                        name = self._template.format(**k)
-                        self.Member().insert1(dict(k, name=name), ignore_extra_fields=True)
+        for scan in StaticMultiDatasetGroupAssignment.fetch(as_dict=True):
+            # Check if the scan has been added to StaticMultiDataset.Member, if not then do it
+            if len(self & dict(group_id = scan['group_id'])) == 0:
+                # Group id has not been added into StaticMultiDataset, thus add it
+                self.insert1(dict(group_id = scan['group_id'], description = scan['description']))
```

Comment:
to keep the consistency with previous descriptions, maybe we can do the '{animal_id}-{'session'}-{scan_idx}' thing here rather than using the description from StaticMultiDatasetGroupAssignment. In fact, not sure if we need two descriptions, if we have it in StaticMultiDatasetGroupAssignment  there is no need to have another description here

Reply:
This was meant to copy whatever new description from StaticMultiDatasetGroupAssignment into the old paradigm to maintain the old legacy code. Edgar already recommend me to switch to a hash which I probably get to it later.